app/waf/src/controllers/common.ts

```diff
@@ -0,0 +1,17 @@
+import {Entity} from '@loopback/repository';
+
+export class Response {
+  [key: string]: Entity;
+
+  constructor(t: typeof Entity, data: Entity) {
+    this[t.modelName.toLowerCase()] = data;
+  }
+}
+
+export class CollectionResponse {
+  [key: string]: Entity[];
+
+  constructor(t: typeof Entity, data: Entity[]) {
+    this[t.modelName.toLowerCase() + 's'] = data;
```

Comment:
https://www.npmjs.com/search?q=plural

Reply:
It is an interesting lib, but it will eventually call " + 's' ", too. It seems no value to utilize it.

Reply:
Done